### DISCUSSION TITLE
@types/gulp-javascript-obfuscator

### DISCUSSION THREAD

**Discussion by Dulux-Oz** [DefinitelyTyped/DefinitelyTyped#58443] [Request a new @types package] [answered]
Hi All,

Could someone help a (TypeScript) noob (again :-) ) with some advice, please, by answering one (or more) of the following questions:

1. Does anyone have the TypeScript def file for gulp-javascript-obfuscator, and may I please have a copy?
2. Could someone please write the TypeScript def file for gulp-javascript-obfuscator?
3. Could someone help me to write the TypeScript def file for gulp-javascript-obfuscator?
4. Could someone point me towards a different JS Obfuscator that works with Gulp and which is TypeScript-useable?
5. Could someone point me towards how to use gulp-javascript-obfuscator "as it is" with TypeScript (or javascript-obfuscator for that matter)?

(FTR: I looked on NPM but couldn't find anything "newer" or "more appropriate" - javascript-obfuscator does have internal TS defs but I think I need to use this gulp plugin as well. Please correct me if I'm wrong.)

Thanks in advance (& thanks again for the previous help I've been given - I really do appreciate it),

Dulux-Oz

**Comment by MysteryBlokHed** [comment, marked as answer]
I just opened #58480

**Reply by MysteryBlokHed** [reply]
Just got merged a few hours ago

**Reply by Dulux-Oz** [reply]
Thanks Mystery  :-)
